readme.md

```diff
@@ -1,4 +1,4 @@
-# 🛶 CANU v1.1.1
+# 🛶 CANU v1.1.3
```

Comment:
Are new features added?  You've incremented bugfix number only and typically w/ a feature prefer middle number increment...  Leave it for now since both of us should have caught this in the develop PR.  For the future though.